spec/src/main/asciidoc/opentelemetry-apis.adoc

```diff
@@ -0,0 +1,90 @@
+
+=== Autoconfigure SPI
+This is the programmatic interface that allows users to register extensions when using the SDK Autoconfigure Extension (which we use for configuration).
+
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/{otel-java-version}/io/opentelemetry/sdk/autoconfigure/spi/package-summary.html[io.opentelemetry.sdk.autoconfigure.spi]
+
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/{otel-java-version}/io/opentelemetry/sdk/autoconfigure/spi/traces/package-summary.html[io.opentelemetry.sdk.autoconfigure.spi.traces]
+
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/{otel-java-version}/io/opentelemetry/sdk/autoconfigure/spi/metrics/package-summary.html[io.opentelemetry.sdk.autoconfigure.spi.metrics]
+
+The above packages have dependencies on the following packages which MUST be supported to the extent that they are required by the Autoconfigure SPI classes:
+
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/{otel-java-version}/io/opentelemetry/sdk/trace/package-summary.html[io.opentelemetry.sdk.trace]
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/{otel-java-version}/io/opentelemetry/sdk/trace/data/package-summary.html[io.opentelemetry.sdk.trace.data]
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/{otel-java-version}/io/opentelemetry/sdk/trace/export/package-summary.html[io.opentelemetry.sdk.trace.export]
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-trace/{otel-java-version}/io/opentelemetry/sdk/trace/samplers/package-summary.html[io.opentelemetry.sdk.trace.samplers]
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-common/{otel-java-version}/io/opentelemetry/sdk/common/package-summary.html[io.opentelemetry.sdk.common]
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-metrics/{otel-java-version}/io/opentelemetry/sdk/metrics/package-summary.html[io.opentelemetry.sdk.metrics]
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-metrics/{otel-java-version}/io/opentelemetry/sdk/metrics/data/package-summary.html[io.opentelemetry.sdk.metrics.data]
+* https://www.javadoc.io/static/io.opentelemetry/opentelemetry-sdk-metrics/{otel-java-version}/io/opentelemetry/sdk/metrics/export/package-summary.html[io.opentelemetry.sdk.metrics.export]
+
+=== Tracing Annotations
+
+* https://www.javadoc.io/doc/io.opentelemetry.instrumentation/opentelemetry-instrumentation-annotations/latest/io/opentelemetry/instrumentation/annotations/package-summary.html[io.opentelemetry.instrumentation.annotations] (`WithSpan` and `SpanAttribute` only)
+
+=== Semantic Conventions Attributes
+
+[NOTE]
+====
+These packages are not stable and MAY be subject to breaking changes in future releases.
```

Comment:
Actually, 2.0 is out and has some stable attributes, shouldn't we adopt that in the new version, since is a breaking change?

Reply:
+1

Reply:
Sure, see #153, which touches that and we need another issue to change the spec and the TCK. This cannot be in scope of this PR.